README.md

```diff
@@ -135,6 +135,8 @@ myNextApplication:
   component: serverless-next.js
   inputs:
     cloudfront:
+      # if you want to use an existing cloudfront distribution, provide it here
```

Comment:
should this be further improved?

Reply:
Yeah maybe provide a link to how "updating" existing distributions works. (https://github.com/serverless-nextjs/serverless-next.js/tree/master/packages/serverless-components/aws-cloudfront#updating-an-existing-cloudfront-distribution)